lib/Serverless.js

```diff
@@ -70,6 +71,12 @@ class Serverless {
       return this.pluginManager.run(this.processedInput.commands);
     }
 
+    // check if tracking is enabled (and track if it's enabled)
+    const serverlessPath = this.config.serverlessPath;
```

Comment:
Yup great point. On the other hand this happens after we're running the command, so might not be that important after all.